src/i_flmusic.c

```diff
@@ -182,7 +182,7 @@ static boolean IsMid(const byte *mem, int len)
 
 static void *I_FL_RegisterSong(void *data, int len)
 {
-    int result = 0;
+    int result = FLUID_FAILED;
```

Comment:
Does this (and probably others if the requested changes) apply to the Woof implementation as well @rfomin ?

Reply:
Oh my, I didn't even see you already applied that. I'm getting old... 😆